algorithms/registration-toolbox/include/registration-toolbox/alignment/base-alignment.h

```diff
@@ -25,11 +25,7 @@ class BaseAlignment {
 
   constexpr aslam::Transformation convertEigenToKindr(
       const Eigen::Matrix4d& T) const {
-    if (!aslam::Transformation::Rotation::isValidRotationMatrix(
```

Comment:
this check seemed to be less conservative than some other checks later down the pipeline, eventually leading to crashes. Enabled renormalization by default since it does not cost much

Reply:
ok, sounds good